### ISSUE TITLE
Make stats persistent in memory

### ISSUE THREAD

**Issue by ShacharHarshuv** [ShacharHarshuv/open-ear#107]
So when you close the app and reopen it it remembers the amount of correct / incorrect questions. (Until you click "reset statistics")

**Comment by entomologe** [comment]
I'd also like to suggest the feature of a simple statistic overview above that option of "reset statistic" in each exercise where the individual percentage on the answers given are listed (i.e. one might be interested which progressions, intervals or chords one has difficulties with). 

I'm looking forward to include it myself but java isn't my preferred language and I never wrote an app for Android. : )

**Comment by ShacharHarshuv** [comment]
> I'd also like to suggest the feature of a simple statistic overview above that option of "reset statistic" in each exercise where the individual percentage on the answers given are listed (i.e. one might be interested which progressions, intervals or chords one has difficulties with).

Yeah that could be a good idea. Can you create a separate issue for this? 

> I'm looking forward to include it myself but java isn't my preferred language and I never wrote an app for Android. : )

This app is not written in java, it's a webapp in Typescript (Javascript) using the Angular framework (which you probably need to know very little about if you don't add a lot of complex UI) with the ionic library (which gives the look of a native app). Feel free to reach out if you need technical help.
